pkg/resource/secret.go

```diff
@@ -61,11 +61,25 @@ func (gs *generatorSecret) expected() (runtime.Object, error) {
 		},
 	}
 
-	data, err := gs.driver.Secrets()
+	configenv, err := gs.driver.ConfigEnv()
 	if err != nil {
 		return nil, err
 	}
 
+	data, err := configenv.SecretData()
+	if err != nil {
+		return nil, err
+	}
+
+	_, _, volumesData, err := gs.driver.Volumes()
```

Comment:
I would still prefer to have the old `Secrets()` function instead of adding a forth return(`volumeData`) to `Volumes()`. What do you reckon?